pipx/commands.py

```diff
@@ -327,6 +335,71 @@ def install(
         venv.remove_venv()
         raise
 
+    # if all is well, write out pipxrc file
+    package_or_url_pipxrc = _abs_path_if_local(package_or_url, venv, pip_args)
+    pipxrc = Pipxrc(venv_dir, read=False)
+    pipxrc.set_package_or_url(package_or_url_pipxrc)
+    pipxrc.set_install_options(pip_args, venv_args, include_dependencies)
+    pipxrc.set_venv_metadata(venv.get_venv_metadata_for_package(package))
+    pipxrc.write()
+
+
+def _abs_path_if_local(package_or_url: str, venv: Venv, pip_args: List[str]) -> str:
+    # TODO 20191001 if not pip --editable, then pip assumes any one-word word
```

Comment:
TODO keyword usually is followed by a ``:``

Reply:
Changed that.